src/components/quantity-counter/QuantitiyCounter.jsx

```diff
@@ -0,0 +1,42 @@
+import AddRoundedIcon from '@mui/icons-material/AddRounded';
+import RemoveRoundedIcon from '@mui/icons-material/RemoveRounded';
+import styles from './QuantitiyCounter.module.css';
+import { useEffect, useState } from 'react';
+
+export const QuantityCounter = ({quantityValue = 0}) => {
+    const [value, setValue] = useState(quantityValue);
+
+    const increaseHandler = () => {
+        setValue(value + 1);
+    }
+
+    const decreaseHandler = () => {
+        setValue(value - 1);
```

Comment:
should be like this ```setValue((value) => value + 1)``` this way it's safer and guaranties that the value is the latest value instead of using it from the top value variable